app/app.go

```diff
@@ -212,6 +228,15 @@ func NewIrisApp(
 	legacyAmino := encodingConfig.Amino
 	interfaceRegistry := encodingConfig.InterfaceRegistry
 
+	// Setup Mempool
+	baseAppOptions = append(baseAppOptions, func(app *baseapp.BaseApp) {
+		memPool := mempool.NoOpMempool{}
+		app.SetMempool(memPool)
+		handler := baseapp.NewDefaultProposalHandler(memPool, app)
+		app.SetPrepareProposal(handler.PrepareProposalHandler())
+		app.SetProcessProposal(handler.ProcessProposalHandler())
+	})
+
```

Comment:
`
func (app *baseapp.BaseApp) {
	memPool := mempool.NoOpMempool{}
	app.SetMempool(memPool)
	handler := baseapp.NewDefaultProposalHandler(memPool, app)
	app.SetPrepareProposal(handler.PrepareProposalHandler())
	app.SetProcessProposal(handler.ProcessProposalHandler())
}
`
Refined into methods to increase readability